tests/testfiles/CommonMark_0.30/entity_and_numeric_character_references/007.test

```diff
@@ -1,17 +0,0 @@
-%   ---RESULT--- "example": 31,
-%   
-%   <a href="&ouml;&ouml;.html">
-%   
-%   ---\RESULT---
-
-\def\markdownOptionHtml{true}
-<<<
-<a href="&ouml;&ouml;.html">
->>>
-documentBegin
-BEGIN link
-- label: 
-- URI: öö.html
-- title: 
-END link
```

Comment:
We currently don't translate HTML elements to their corresponding markdown equivalents.